### DISCUSSION TITLE
Docs Discussion

### DISCUSSION THREAD

**Discussion by Sddilora** [octoposprime/op-docs#3] [General]
Hi there!

**Comment by Sddilora** [comment]
I want to inquire about project management. Is it possible to archive the resolved issues in the [op-be project](https://github.com/orgs/octoposprime/projects/11/views/1) to make it easier to manage and view the unresolved ones?

**Comment by husamettinarabaci** [comment]
Yes. I have added and saved a filter. You can see only if the issue is open. If you can set any filter and save everybody will be affected.

is:open

**Comment by Sddilora** [comment]
thanks
